src/openrct2/config/Config.h

```diff
@@ -15,6 +15,8 @@
 
 #include <string>
 
+enum class TemperatureFormat : int32_t;
```

Comment:
Seems a bit overkill to have the underlying type as `int32_t` (even though that's what it was previously
```suggestion
enum class TemperatureFormat : bool;
```

I'd go for `bool`

Reply:
I think it was 32 due to it being a config option. There might be issues with changing the size.

Reply:
Compilation issues or running/replay issues? I'm alright if you want to keep it untouched for now

Reply:
Why would you pick a bool? That type has a different purpose and is not used for integers.

Reply:
Would fit just as well, unless we're planning to go for Kelvin eventually :¶

Not too attached, go for the the uint8 suggestion if it works

Reply:
bool is just not logical for enums. I think int32_t will do fine.